apps/server/src/models/eventsDefinition.ts

```diff
@@ -1,10 +1,11 @@
 import { EndAction, OntimeBlock, OntimeDelay, OntimeEvent, SupportedEvent, TimerType } from 'ontime-types';
 
-export const event: Omit<OntimeEvent, 'id' | 'delay' | 'cue'> = {
+export const event: Omit<OntimeEvent, 'id' | 'delay'> = {
   title: '',
   subtitle: '',
   presenter: '',
   note: '',
+  cue: '',
```

Comment:
hmm, the rules of our app declare that we cannot have a empty cue field, so we should not provide a placeholder event which breaks our rules


I know I said we could, but I am not sure I am comfortable with the change. 
the reasoning being, what makes `id` different from `cue`? if we add `cue` should we not add `id`? Since for now our use case is very simple, I think we can postpone this change until we have a better use case for it to be necessary

Do you agree?

Reply:
yes we can wait